requirements.txt

```diff
@@ -1,128 +1,128 @@
-aws-cdk.alexa-ask==v1.114.0
```

Comment:
Are the CDK version bumps here required for any of the pipelines you have at the moment?

We're looking at migrating to CDK V2 in the near-future, so we were planning on keeping this at `v1.114.0` until we get this work scheduled in 👍

Reply:
We should be able to patch the libs we need in the CI pipeline, so we can probably skip the version bump and just include the base image update, sure!

Reply:
Nice one, thanks for that! We'll also be doing a redesign of how dependencies are handled here, again in the near-future - will keep you posted 👍